.travis/osx..install.sh

```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-PACKAGES="cmake pkg-config fftw libogg libvorbis libsndfile libsamplerate jack sdl stk portaudio node fltk"
+PACKAGES="cmake pkgconfig fftw libogg libvorbis lame libsndfile libsamplerate jack sdl libgig libsoundio stk portaudio node fltk"
```

Comment:
It's most likely to using [xcode 8.2](https://github.com/liushuyu/lmms/blob/5ec242bf29859d71cc0554c25f2f89bbcd17ecfd/.travis.yml#L15) since the Travis-CI bug report mentions that multiple times.  We can merge with this failing, it's fine.